src/main/java/pro/cloudnode/smp/nations/commands/NationsCommand.java

```diff
@@ -1,22 +1,422 @@
+            sendMessage(t(Messages.COMMANDS_ITEM, "list", "", "list all nations"));
+            sendMessage(t(Messages.COMMANDS_ITEM, "quit", "", "quit your nation"));
+            sendMessage(t(Messages.COMMANDS_ITEM, "info", "", "get info about your nation"));
+            sendMessage(t(Messages.COMMANDS_ITEM, "option", "<key> <value>", "set options for your nation"));
+            sendMessage(t(Messages.COMMANDS_ITEM, "join", "<nation>", "join a nation"));
+            sendMessage(t(Messages.COMMANDS_ITEM, "help", "", "show this help message"));
+            if (sender.hasPermission("nations.admin")) {
+                sendMessage(t(Messages.COMMANDS_ITEM, "force-delete", "<nation>", "force delete a nation"));
+                sendMessage(t(Messages.COMMANDS_ITEM, "reload", "", "reload the plugin"));
+            }
+            return;
+        }
+
+        switch (args[1]) {
+            case "create":
+                sendMessage(t(Messages.USAGE, label, "create", "<name>"));
+                break;
+            case "invite":
+                sendMessage(t(Messages.USAGE, label, "invite", "<player>"));
+                break;
+            case "kick":
+                sendMessage(t(Messages.USAGE, label, "kick", "<player>"));
+                break;
+            case "list":
+                sendMessage(t(Messages.USAGE, label, "list", ""));
+                break;
+            case "quit":
+                sendMessage(t(Messages.USAGE, label, "quit", ""));
+                break;
+            case "info":
+                sendMessage(t(Messages.USAGE, label, "info", ""));
+                break;
+            case "option":
+                sendMessage(t(Messages.USAGE, label, "option", "<key> <value>"));
+                break;
+            case "join":
+                sendMessage(t(Messages.USAGE, label, "join", "<nation>"));
+                break;
+            default:
+                help(sender, label, new String[0]);
+                break;
+        }
+    }
+
+    private void join(CommandSender sender, String label, String[] args) {
+        if (args.length <= 1) {
+            sendMessage(t(Messages.USAGE, label, "join", "<nation>"));
+            return;
+        }
+
+        if (nationManager.isInNation((Player) sender)) {
+            sendMessage(t(Messages.ALREADY_IN_NATION));
+            return;
+        }
+
+        Nation nation = Nations.getNationManager().get(args[1]);
+        if (nation == null) {
+            sendMessage(t(Messages.NATION_NOT_FOUND, args[1]));
+            return;
+        }
+
+        if (!nation.invited.contains(getPlayer().getUniqueId())) {
+            sendMessage(t(Messages.NOT_INVITED_TO_NATION, nation));
+            return;
+        }
+
+        nation.addMember(getPlayer().getUniqueId());
+        nation.removeInvited(getPlayer().getUniqueId());
+        sendMessage(t(Messages.JOINED_NATION, nation));
```

Comment:
should probably also send message to online nation members or the leader. maybe like `nations.notify.join` permission